public/docs/ts/latest/guide/component-styles.jade

```diff
@@ -156,37 +157,42 @@ a(id='loading-styles')
   ### Style URLs in Metadata
   
   We can load styles from external CSS files by adding a `styleUrls` attribute
-  into a component's `@Component` or `@View` decorator:
+  into a component's `@Component` or `@View` #{_decorator}:
   
 +makeExample('component-styles/ts/app/hero-details.component.ts', 'styleurls')
-  
+
 .alert.is-important
-  :marked
-    The URL is ***relative to the application root*** which is usually the
-    location of the `index.html` web page that hosts the application. 
-    The style file URL is *not* relative to the component file.
-    That's why the example URL begins `app/`.
-    See [Appendix 2](#relative-urls) to specify a URL relative to the component file.
-  
-.l-sub-section
-  :marked
-    Users of module bundlers like Webpack may also use the `styles` attribute to load 
-    styles from external files at build time. They could write:
-    
-    `styles: [require('my.component.css')]`
+  block style-url-alert
+    :marked
+      The URL is ***relative to the application root*** which is usually the
+      location of the `index.html` web page that hosts the application. 
+      The style file URL is *not* relative to the component file.
+      That's why the example URL begins `app/`.
+      See [Appendix 2](#relative-urls) to specify a URL relative to the 
+      component file.
+
+block module-bundlers
+  .l-sub-section
+    :marked
+      Users of module bundlers like Webpack may also use the `styles` attribute
```

Comment:
Right. Whenever you see a line starting with
```jade
block foo
  some-stuff
```
Then look for the correspondingly named block in the Dart `.jade` file.